lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2627,8 +2627,9 @@ ParseResult FIRStmtParser::parseInstance() {
                    });
   auto name = dontTouch ? id : filterUselessName(id);
 
-  auto result = builder.create<InstanceOp>(
-      resultTypes, moduleName, name, annotations.first, annotations.second);
+  auto result =
+      builder.create<InstanceOp>(resultTypes, moduleName, name,
+                                 annotations.first, annotations.second, false);
```

Comment:
isn't false defaulted?

Reply:
I had used this without investigating what's going on here... There's a bug in that builder for [the path where `parseElidePortAnnotations` constructs a default vector of port annotations and this fails an assertion in the builder](https://github.com/llvm/circt/blob/main/lib/Dialect/FIRRTL/FIRRTLOps.cpp#L855).  By using an explicit `false` here this is avoiding that bug by _not_ using that builder. I'll fix this on main first and use the default here.

Reply:
Not a bug. 🤦 Just some confusion over which builder is actually getting called. It looks like `ArrayAttr` can be promoted to `ArrayRef<Attribute>` and introduce another level of nesting if you aren't careful. Fixing this with some explicit `getValue` calls to actually extract the `ArrayRef`.